docs/docs/components/reanimated_swipeable.md

```diff
@@ -81,27 +81,29 @@ an argument.
 
 ### `renderLeftActions`
 
-method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
-This map describes the values to use as inputRange for extra interpolation:
+accepts a function which receives the following arguments:
```

Comment:
a function that returns a component that is going to be rendered under `swipeable` after swiping to the right. The function receives the following arguments:

Reply:
We think that _**under**_ fits better in that case

Reply:
Makes sense, done: [e37a656](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/e37a6568d2f6d939cd873bec95a7d103980d8f12)